rir/src/compiler/pir/pir.h

```diff
@@ -1,6 +1,18 @@
 #ifndef PIR_PIR_H
 #define PIR_PIR_H
 
+// clang-format off
+//********** VERBOSE BIT MASK ***********
+#define PRINT_ORIGINAL_MASK   0x1  // bit 1: RIR Original version
```

Comment:
Sorry to keep nitpicking this, but I prefer to avoid C-style macros when possible. I'd suggest turning these to consts, e.g.

`const uint8_t PrintOriginalMask = 0x1`

(I guess we can't really use enum classes unless you override all the bitwise operators. Probably not worth the trouble.)